controllers/gitopsservice_controller_test.go

```diff
@@ -303,7 +334,7 @@ func TestReconcile_consoleAPINotFound(t *testing.T) {
 	s := scheme.Scheme
 	addKnownTypesToScheme(s)
 
-	fakeClient := fake.NewFakeClient(newGitopsService())
+	fakeClient := fake.NewFakeClient(util.NewClusterVersion("4.15.1"), newGitopsService())
```

Comment:
@ciiay is this required here?

Reply:
@jaideepr97 Yes, it's needed, because I’m checking the OCP version in Reconcile function, if I don’t set it in the unit tests, it will through error and unit tests will fail. Or is there a better way to solve the problem?

Reply:
please see https://github.com/redhat-developer/gitops-operator/pull/398/files#r1029254556